server/channels/api4/report.go

```diff
@@ -83,7 +83,7 @@ func startUsersBatchExport(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	startAt, endAt := model.GetReportDateRange(r.URL.Query().Get("date_range"), time.Now())
```

Comment:
Looking at the implementation of `GetReportDateRange`, specifying the `date_range` params looks mandatory. If I call the `localhost:8065/api/v4/reports/users/export` API without any query param, the API call is successful(response 200 OK),  but report's start and end time turns out to be 0 and 0, and the worker immediately fails with the error -

```json
{
	"timestamp": "2024-01-25 09:35:56.478 +05:30",
	"level": "error",
	"msg": "Worker: Failed to get data for report batch. Exiting",
	"caller": "jobs/batch_report_worker.go:55",
	"error": "failed to parse job metadata: strconv.ParseInt: parsing \"\": invalid syntax"
}
```

We either need to seta. default value to this (and any other) param, or validate the API inputs in API layer itself and return appropriate error response.

Reply:
For example, the function `getTranslatedDateRange` we're using the default value of time period as "all time". I'f we're making a time period default, can we make it last 30 days so as to not accidentally trigger a humongous job?